### PR TITLE
navigation atoms

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
Navigation atoms created

### CODE REVIEW

File: stories/Atom/Navigation/Breadcrumb/Breadcrumb.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './breadcrumb.scss';
+
+export const Breadcrumb = ({ style, text, ...props }) => {
+  return (
+    <ul className={['breadcrumb',`breadcrumb--${style}`].join(' ')}   {...props}>
+      <li><a href="#">{text}</a></li>
+    </ul>
```
Comment (surbhi90) [inline stories/Atom/Navigation/Breadcrumb/Breadcrumb.jsx:9]: @harishkumar-srijan We should not use the complete ul tag for breadcrumb item

File: stories/Atom/Navigation/Breadcrumb/Breadcrumb.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import { Meta, Story, ArgsTable } from '@storybook/addon-docs';
```
Comment (surbhi90) [inline stories/Atom/Navigation/Breadcrumb/Breadcrumb.stories.mdx:1]: @harishkumar-srijan Remove ArgsTable

File: stories/Atom/Navigation/Language-switcher-row/LanguageSwitcherRow.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export const LanguageSwitcherRow = ({ color, text, ...props }) => {
+  return (
+     <div className="language_item">{text}</div>
+  );
+};
+
+LanguageSwitcherRow.propTypes = {
+  color: PropTypes.oneOf(['Red', 'White' ])
+};
```
Comment (surbhi90) [inline stories/Atom/Navigation/Language-switcher-row/LanguageSwitcherRow.jsx:13]: @harishkumar-srijan Do we have red and white variants of languageswitcher

File: stories/Atom/Navigation/Language-switcher-row/LanguageSwitcherRow.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+
+// * language row start *//
+.language_item {
+  @extend %paragragh-font-16;
+  @extend %border-bottom;
+  padding: 12px 16px;
+  max-width: 100px;
```
Comment (surbhi90) [inline stories/Atom/Navigation/Language-switcher-row/LanguageSwitcherRow.scss:10]: @harishkumar-srijan use min-width instead max-width

File: stories/Atom/Navigation/Loader/loader.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+
+// * loader start *//
+.loader {
+  border-radius: 50%;
+  border: 8px solid $color-blue-600;
+  border-top: 8px solid #f3f3f3;
```
Comment (surbhi90) [inline stories/Atom/Navigation/Loader/loader.scss:9]: @harishkumar-srijan use the variable

File: stories/Atom/Navigation/Loader/loader.scss [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  width: 40px;
+  height: 40px;
+  @include loader;
+  @include devicebreak(small) {
+    width: 96px;
+    height: 96px;
+    border: 14px solid $color-blue-600;
+    border-top: 14px solid #f3f3f3;
```
Comment (surbhi90) [inline stories/Atom/Navigation/Loader/loader.scss:17]: @harishkumar-srijan use variable

File: stories/Atom/Navigation/Loader/loader.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    margin-top: 40px;
+  }  
+`}</style>
+
+# Loader Component
+
+The loader component is used to show the loading time of the web page. It will be animated until the complete site loading.
+
+<Story name="Loader">
+  {(args, { globals: { locale } }) => {
```
Comment (surbhi90) [inline stories/Atom/Navigation/Loader/loader.stories.mdx:23]: @harishkumar-srijan remove args, globals

File: stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+// * progress navigation start *//
+.progress-container {
+  width: 100%;
+  height: 12px;
+  background: $color-gray-300;
+  @include devicebreak(small) {
+   height: 16px;
+  }
+  .progress-bar {
```
Comment (surbhi90) [inline stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss:13]: @harishkumar-srijan can we give this styling outside the progress-container

File: stories/Atom/Navigation/Sidebar-second-level/sidebarsecondlevel.scss [hunk 9]
```diff
@@ -0,0 +1,16 @@
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/breakpoints";
+@import "../../../assets/scss/variables";
+
+// * Slidebar Second level start *//
+.sidebar_item__sub {
+	padding: 16px 24px 16px 32px;
+	@extend %paragragh-font-16;
+	@include devicebreak(small) {
+		padding: 16px 24px 16px 48px;
```
Comment (surbhi90) [inline stories/Atom/Navigation/Sidebar-second-level/sidebarsecondlevel.scss:10]: @harishkumar-srijan only update the padding-left here

File: stories/Atom/Navigation/Sidebar-with-arrow/SidebarWithArrow.scss [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+	}
+	h6 {
+		margin-bottom: 0;
+	}
+	.arrow {
+		background: url(#{$img-path-Icon}//Chevron-down.svg) no-repeat center center;
+		float: right;
+		width: 20px;
+		height: 20px;
+		border-top-color: #f3f3f3;
```
Comment (surbhi90) [inline stories/Atom/Navigation/Sidebar-with-arrow/SidebarWithArrow.scss:36]: @harishkumar-srijan use variable

File: stories/Atom/Navigation/Sidebar-with-arrow/SidebarWithArrow.scss [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+		}
+		&:hover {
+			background-color: $color-blue-600!important;
+		}
+		.arrow {
+			background: url(#{$img-path-Icon}//Chevron-down-white.svg) no-repeat center center;
+			transform: rotateZ(180deg);
+		}
+	}
+
```
Comment (surbhi90) [inline stories/Atom/Navigation/Sidebar-with-arrow/SidebarWithArrow.scss:71]: @harishkumar-srijan selected and selected-up have same styling so can we create placeholder

File: stories/Atom/Navigation/Sidebar-with-arrow/SidebarWithArrow.scss [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+			background: url(#{$img-path-Icon}//Chevron-down-white.svg) no-repeat center center;
+		}
+	}
+	&--Selected-up {
+		background-color: $color-blue-600;
+		h6 {
+			color: $color-white;
+		}
+		&:hover {
+			background-color: $color-blue-600!important;
```
Comment (surbhi90) [inline stories/Atom/Navigation/Sidebar-with-arrow/SidebarWithArrow.scss:64]: @harishkumar-srijan don't use !important

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by harishkumar-srijan** [comment]
old PR, New PR merged and in the branch, we facing the style.scss, style.map issue in this PR

**Comment by surbhi90** [comment]
Due to conflicts, closing this PR

**Pull request closed without merging**